src/Geometry/coordinates.jl

```diff
@@ -129,6 +133,10 @@ product_coordinates(yp::YPoint, zp::ZPoint) = YZPoint(promote(yp.y, zp.z)...)
 product_coordinates(xyp::XYPoint, zp::ZPoint) =
     XYZPoint(promote(xyp.x, xyp.y, zp.z)...)
 
+product_coordinates(latp::LatPoint, longp::LongPoint) =
+    LatLongPoint(promote(latp.lat, longp.long)...)
+product_coordinates(longp::LongPoint, latp::LatPoint) =
```

Comment:
Does `promote` preserve the order? it looks like the order is swapped here, does that matter?